### ISSUE TITLE
Better Confirmation Request Has Been Sent

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#45]
After a request has been sent it takes a few minutes for it to appear in search results on the site, or on a users' "my requests" page. 

This results in users wondering if it has got lost (and writing to us to ask) or in some cases making the same request again. 

Ideas:
- A warning that the request might not appear in searches / on the user page for a while. 
- Fixing the slow search indexing problem (even if just for the relevant user and body pages)
- Providing users a link to their new request; perhaps in a flash message; perhaps even by email (for their records)

**Comment by itsaphel** [comment]
I think I ran into this issue. It seems like users are shown their new request and emailed a link immediately, but the search indexes don't update for a while. I figured it was a bug on my end/installation.

Not sure but maybe it's related to https://github.com/mysociety/alaveteli/blob/develop/app/controllers/public_body_controller.rb#L63 ? (I recall rebuilding the indexes using `bundle exec script/destroy-and-rebuild-xapian-index` was the only way to get the new request to show up on my install)
